docs/docs/08-guides/03-react-vite-websockets.md

```diff
@@ -101,7 +101,7 @@ Now, we will create our backend for our app:
 
     ```sh
     npm install -g winglang
-    wing -V # should be >= 0.60.1
```

Comment:
`-V` should be removed from the CLI code (even though it doesn't work anymore anyways)

Also, -V is used in the benchmark tests https://github.com/winglang/wing/blob/main/tools/hangar/src/benchmarking/cli.bench.ts#L18

Reply:
Is there a way to support both `-v` and `-V`? (if this has already been proposed feel free to ignore 😅)

Reply:
I couldn't find support for this documented in commander, nor any issues mentioning it.
We could probably hack it together, but imo -v seems more common and is probably sufficient.